meta/meta.go

```diff
@@ -144,6 +153,21 @@ func (m *Meta) GetGlobalID() (int64, error) {
 	return m.txn.GetInt64(mNextGlobalIDKey)
 }
 
+func (m *Meta) GenPolicyID() (int64, error) {
```

Comment:
Maybe generate the id when creating the new policy? a.k.a auto incremental id.

Reply:
It's

Reply:
@AilinKid Actually, I mean "remove this API". `GenPolicyID` is used outside `CreatePolicy`, the behavior is not controlled by `meta`. One could just generate a random number other than numbers from `GenPolicyID`. Is there any advantage with a standalone ID generating method? Otherwise I prefer less/simple/opinionated API.

**Not a necessary change request. rest code LGTM**

Reply:
Done